bin/smoke

```diff
@@ -40,7 +40,7 @@ if [[ $tests == *kotlin* ]]
 then
   cd kotlin || exit
   echo "Smoking Kotlin SDK ..."
-  gradle test
```

Comment:
Random non-Swift file changes above here... needed?

Reply:
Yeah, just a minor smoke test fix for a smoke test requiring less setup for Kotlin